packages/cron/src/lib/email.js

```diff
@@ -0,0 +1,89 @@
+import debug from 'debug'
+
+export const emailType = {
+  Used75PercentStorage: 'Used75PercentStorage',
+  Used80PercentStorage: 'Used80PercentStorage',
+  Used85PercentStorage: 'Used85PercentStorage',
+  Used90PercentStorage: 'Used90PercentStorage',
+  UsedOver100PercentStorage: 'UsedOver100PercentStorage'
+}
+
+const log = debug('email:EmailService')
+
+export class EmailService {
```

Comment:
Ignoring this for now, since it's likely going to be tweaked and updated by @adamalton